src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -196,7 +198,18 @@ const formConfig = {
           schema: {
             type: 'object',
             properties: {
-              servicePeriods
+              servicePeriods: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    dateRange: _.set('required', ['from'], dateRange),
```

Comment:
This requires that dateRange has a 'from' property to be valid.

Reply:
Seems like this should be set in the schema, but this is probably okay. :man_shrugging:

Reply:
Schema sets both `from` and `to` as required, which matches `submit` swagger so should be used in the config. Also, just as FYI, I'm working on some changes that restrict the `serviceBranch` to an `enum` list per EVSS. But that's also getting added to the schema, so shouldn't require anything else here as long as the schema's version of the definition is used.

Reply:
Thanks @markgreenburg will update using the schema.